docs/guides/tutorials/phoenix_github_oauth.rst

```diff
@@ -314,7 +314,7 @@ This part will be pretty big, as we'll talk about using ``Ecto.Changeset``
 with the EdgeDB driver, as well as modules and queries related to user
 registration via GitHub OAuth.
 
-``Ecto`` provides ``Ecto.Changeset``s, which are convenient to use when
+``Ecto`` provides ``Ecto.Changeset`` s, which are convenient to use when
 working with ``Ecto.Schema`` to validate external parameters and we can use
 them also using ``EdgeDBEcto``, though not quite as fully as we can with the
 full-featured adapters for ``Ecto``.
```

Comment:
```suggestion
``Ecto`` provides "changesets" (via ``Ecto.Changeset``), which are convenient
to use when working with ``Ecto.Schema`` to validate external parameters. We
could use them via ``EdgeDBEcto`` instead, though not quite as fully as we can
with the full-featured adapters for ``Ecto``.
```

Trying to avoid the dangling "s" and also break this long sentence up a bit. I'm not totally confident I understood the original, so please check my understanding before you accept.